### DISCUSSION TITLE
Add way to configure redirect to /admin

### DISCUSSION THREAD

**Discussion by scouech** [directus/directus#4607] [Feature Requests]
👋

To access Directus I can either go to `DOMAIN/admin/` or `DOMAIN` and I will be redirected to `DOMAIN/admin/`

Is there a way to stop `DOMAIN` redirecting to `/admin/` ?

Cheers!

**Comment by rijkvanzanten** [comment]
There's currently no setting / way to do that in Directus directly

**Reply by scouech** [reply]
any way to do it server side? by modifying a file or something?

**Reply by rijkvanzanten** [reply]
hmmmmmmmmmmm the redirect is hardcoded in here:

https://github.com/directus/directus/blob/f2bc53ca84a9868f3861b770bbfb25ac458b26ec/api/src/app.ts#L113

so you can't just undo that. I guess you could probably set a manual redirect in a webserver in front of Directus 🤔  (Assuming you're using Nginx/Apache/Caddy as a reverse proxy in front of Directus)

Alternatively, you could open a PR that introduces a `ROOT_REDIRECT` environment variable to configure this 😄

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
